src/language/__tests__/printer-test.js

```diff
@@ -174,4 +178,45 @@ describe('Printer: Query document', () => {
     `),
     );
   });
+
+  it('keeps arguments on one line if line is short (<= 80 chars)', () => {
+    const printed = print(
+      parse('{trip(wheelchair:false arriveBy:false){dateTime}}'),
+    );
+
+    expect(printed).to.equal(
+      // $FlowFixMe[incompatible-call]
+      dedent(String.raw`
+      {
+        trip(wheelchair: false, arriveBy: false) {
+          dateTime
+        }
+      }
+    `),
+    );
+  });
+
+  it('puts arguments on multiple lines if line is long (> 80 chars)', () => {
+    const printed = print(
+      parse(
+        '{trip(wheelchair:false arriveBy:false includePlannedCancellations:true transitDistanceReluctance:2000){dateTime}}',
+      ),
+    );
+
+    expect(printed).to.equal(
+      // $FlowFixMe[incompatible-call]
+      dedent(String.raw`
```

Comment:
You can use `dedent` without `String.raw`.

Reply:
✅